tools/global_bss_check.py

```diff
@@ -142,63 +123,37 @@ def main():
     """
 
     parser = argparse.ArgumentParser(description=description, epilog=epilog, formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument("mapFile", help="Path to a map file.")
-    parser.add_argument("mapFileExpected", help="Path to the expected map file. Optional, default is 'expected/mapFile'.", nargs="?", default="")
     parser.add_argument("-a", "--print-all", help="Print all bss, not just non-matching.", action="store_true")
     parser.add_argument("-n", "--no-fun-allowed", help="Remove amusing messages.", action="store_true")
+    parser.add_argument("-r", "--no-reverse-check", help="Disable looking for symbols on the expected map that are missing on the built map file.", action="store_true")
+    parser.add_argument("-s", "--section", help="Specify a section other than bss to check for reordered symbols", default=".bss")
     args = parser.parse_args()
 
-    if args.mapFileExpected == "":
-        args.mapFileExpected = os.path.join("expected", args.mapFile)
-
-    badFiles, missingFiles, comparedDict = compareMapFiles(args.mapFile, args.mapFileExpected)
-    printCsv(badFiles, missingFiles, comparedDict, args.print_all)
-
-    if len(badFiles) + len(missingFiles) != 0:
-        print("", file=os.sys.stderr)
-
-        if len(badFiles) != 0:
-            print(colorama.Fore.RED + "  BAD" + colorama.Style.RESET_ALL)
-
-            for file in badFiles:
-                print(f"bss reordering in {file}", file=os.sys.stderr)
-            print("", file=os.sys.stderr)
-            
-            if not args.no_fun_allowed:
-                print(colorama.Fore.LIGHTWHITE_EX +
-                "  BSS is REORDERED!!\n"
-                "  Oh! MY GOD!!" 
-                + colorama.Style.RESET_ALL, file=os.sys.stderr)
-                print("", file=os.sys.stderr)
+    mapfilePath = Path("build/mm.map")
```

Comment:
More so note to self. Uses mapfile so would need to be updated when the build sync PR gets in (or vice versa if this gets in first).

Reply:
Bump. Mapfile path will now need to be updated.